src/pages/welcome.astro

```diff
@@ -277,6 +276,10 @@ changeLanguage('en')
         text-align: center;
       }
 
+      .heroContent img {
+        display: none;
+      }
+
```

Comment:
Why not showing the default loading image?
Also, consider using `visibility: hidden` so the image load doesn't cause layout shift.